lib/project_types/script/layers/infrastructure/script_repository.rb

```diff
@@ -8,12 +8,11 @@ class ScriptRepository
         property! :ctx, accepts: ShopifyCli::Context
 
         def get_script(language, extension_point_type, script_name)
-          source_file_path = src_code_file(language)
-          unless ctx.file_exist?(source_file_path)
-            raise Domain::Errors::ScriptNotFoundError.new(extension_point_type, source_file_path)
+          unless ctx.file_exist?(src_code_file)
```

Comment:
This `script_repository` will be deleted but I will do that in another PR to reduce PR bloat. The changes here are just so that existing scripts will still work. Basically since this repo depends on the structure of a script, it prevents us from being language agnostic. The only things we really need from the Script entity are `EP` and `ScriptName`, which are both available from the `ScriptProject`.